renderer/src/components/CharacterKillsCard.jsx

```
@@ -184,93 +254,73 @@
-              <p>
-                Aggregated <span className="font-medium">{aggregationInfo.period_type}</span> stats for <span className="font-medium">{aggregationInfo.target_date}</span>
-              </p>
-            </div>
-          </div>
-        )}
+          <button
+            onClick={fetchDebugData}
+            disabled={loading}
+            title="View Database Status"
+            className="p-3 bg-gray-600 text-white rounded-lg hover:bg-gray-700 transition flex items-center justify-center disabled:opacity-50"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)